isomorphe/templates/select.html.j2

```diff
@@ -136,7 +136,11 @@
                     hx-post="{{ url_for("select_preview") }}"
                     hx-target="#select_preview"
                     hx-swap="innerHTML show:top"
-                    class="fr-btn fr-btn--secondary fr-mr-1w">Prévisualiser la recherche</button>
+                    hx-indicator=".spinner-indicator"
```

Comment:
```suggestion
                    hx-indicator=".spinner-indicator"
                    hx-disabled-elt="this"
```

This will also disable the button while loading. Up to you.

Reply:
Cool! How about disabling all buttons?
```
hx-disabled-elt=".fr-btn"
```

Reply:
Why not, seems to make sense for the 3 buttons we have.

Reply:
Agreed, we can rescope it if/when we add more buttons.